### PR TITLE
kde-plasma/plasma-meta: add mediacenter and sdk useflags

### PR DESCRIPTION
**Pull request by Vindex17**
I think it's better this way because it's more modular (mediacenter it's not something everyone needs)

### CODE REVIEW

**Comment by kensington** [comment]
Thanks, this looks good. Just needs one small is needed: see check output:

    IUSE.invalid [fatal]          1

    kde-plasma/plasma-meta/plasma-meta-5.4.1.ebuild: mediacenter

Since mediacenter is not a global USE flag, it needs to be added to this package's metadata.xml.

**Comment by Vindex17** [comment]
Ok. Just another couple of question before proceeding:
1) Do I have to apply it to git packages too?
2) Can I do the same thing with plasma-sdk dep?

**Comment by kensington** [comment]
1. Yep, please apply to git packages too. We use these ebuilds as the basis for all version bumps, so any changes not applied there will end up lost.
2. OK with me.

File: kde-plasma/plasma-meta/plasma-meta-5.4.49.9999.ebuild [hunk 1]
```diff
@@ -11,8 +11,8 @@ HOMEPAGE="https://www.kde.org/workspaces/plasmadesktop/"
 
 LICENSE="metapackage"
 SLOT="5"
-KEYWORDS=""
-IUSE="bluetooth +display-manager gtk +sddm networkmanager pulseaudio +wallpapers"
+KEYWORDS="~amd64"
```
Comment (kensington) [inline kde-plasma/plasma-meta/plasma-meta-5.4.49.9999.ebuild]: Oops, keywords don't belong on live ebuilds.

**Comment by kensington** [comment]
Thanks, rebased and merged as 782162b3ada9c1092f5b944380f698c2b420c668.

**Pull request closed without merging**